AlwaysOnKotlin/gradle/libs.versions.toml

```diff
@@ -1,21 +1,30 @@
 [versions]
 androidx-activity = "1.4.0"
-compose-ui = "1.1.0-rc01"
 androidx-test = "1.4.0"
-androidx-wear-compose = "1.0.0-alpha13"
+androidx-wear-compose = "1.0.0-alpha16"
+compose = "1.2.0-alpha03"
 org-jetbrains-kotlinx = "1.6.0"
 
 [libraries]
+android-build-gradle = "com.android.tools.build:gradle:7.1.1"
+android-lint-gradle = "com.android.tools.lint:lint-gradle:30.1.1"
```

Comment:
This seems stricter, from CI failures.

```
> Task :views:lintDebug FAILED
/home/runner/work/wear-os-samples/wear-os-samples/AlwaysOnKotlin/views/src/main/AndroidManifest.xml:43: Error: Redundant label can be removed [RedundantLabel]
            android:label="@string/app_name"
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Yeah, but it's even better. Should be fixed now